src/sentry/grouping/enhancer/actions.py

```diff
@@ -144,7 +149,8 @@ def __init__(self, var, value):
         self._is_updater = self.var not in VarAction._FRAME_VARIABLES
 
         try:
-            self.value = VarAction._VALUE_PARSERS[var](value)
+            # error: "object" not callable  [operator]
+            self.value = VarAction._VALUE_PARSERS[var](value)  # type: ignore[operator]
```

Comment:
this can be fixed with this above:

```python
_VALUE_PARSERS: dict[str, Callable[[Any], Any]] = {
```

kinda unfortunate that this would still keep the resulting values "untyped" but it would at least make this call checked properly